app/javascript/dashboard/routes/dashboard/helpcenter/constants.js

```diff
@@ -0,0 +1,7 @@
+export default {
```

Comment:
Please use the main constants file https://github.com/chatwoot/chatwoot/blob/develop/app/javascript/dashboard/constants.js
```
import wootConstants from 'dashboard/constants';
const {
    ARTICLE_STATUS_TYPES
} = wootConstants;
```